cluster_experiments/experiment_analysis.py

```diff
@@ -301,6 +302,114 @@ def from_config(cls, config):
         )
 
 
+class PairedTTestClusteredAnalysis(ExperimentAnalysis):
+    """
+    Class to run paired T-test analysis on aggregated data
+
+    Arguments:
+        cluster_cols: list of columns to use as clusters
+        target_col: name of the column containing the variable to measure
+        treatment_col: name of the column containing the treatment variable
+        treatment: name of the treatment to use as the treated group
+        strata_cols: list of index columns for paired t test. Should be a subset or equal to cluster_cols
+
+    Usage:
+
+    ```python
+    from cluster_experiments.experiment_analysis import PairedTTestClusteredAnalysis
+    import pandas as pd
+
+    df = pd.DataFrame({
+        'x': [1, 2, 3, 4, 0, 0, 1, 1],
+        'treatment': ["A", "B", "A", "B"] * 2,
+        'cluster': [1, 2, 3, 4, 1, 2, 3, 4],
+    })
+
+    PairedTTestClusteredAnalysis(
+        cluster_cols='cluster',
+        strata_cols='cluster',
+        target_col=df['x'],
```

Comment:
ah I know why, you have to add your class in tests/test_docs, and this test will fail. Could you also add in in the readme, in the `__init__`, and increase the library version in setup.py to 0.5.1?